charts/verdaccio/values.yaml

```diff
@@ -2,6 +2,8 @@ image:
   repository: verdaccio/verdaccio
   tag: 4.6.2
   pullPolicy: IfNotPresent
+  pullSecrets:
+    # name: dockerhub-secret
```

Comment:
ahh I see
https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-pod-that-uses-your-secret

👍